packages/network-of-terms-catalog/catalog/datasets/filmmuseum-genres.rq

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.beeldengeluid.nl/gtaa/Genre_Filmmuseum",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Filmmuseum: Genres"
+    }
+  ],
+  "creator": [
+    {
+      "@id": "https://www.beeldengeluid.nl/"
+    }
+  ],
+  "url": [
+    "http://data.beeldengeluid.nl/gtaa/Genre_Filmmuseum"
+  ],
+  "distribution": [
+    {
+      "@id": "https://data.beeldengeluid.nl/id/datadownload/0033",
+      "@type": "DataDownload",
+      "contentUrl": "https://gtaa.apis.beeldengeluid.nl/sparql",
+      "encodingFormat": "application/sparql-query",
+      "potentialAction": [
+        {
+          "@type": "SearchAction",
+          "query": "file://catalog/queries/search/genre-filmmuseum.rq"

```

Comment:
The actual file names still have underscores, causing the automated tests to fail on GH Actions.

Reply:
they are not in the local branch. I just removed the dataset files that had underscores and pushed. query files with underscores I don't see.